Sources/SwiftSyntax/SyntaxNodes.swift.gyb.template

```diff
@@ -187,21 +187,14 @@ public struct ${node.name}: ${base_type}Protocol, SyntaxHashable {
 %       end
   public var ${child.swift_name}: ${child_type}${optional_mark} {
     get {
```

Comment:
Could go even furtherrrr if you wanted to, ie. get/set on the same line as the body. Or does swift-format not like that?